src/chainstate/burn/operations/mod.rs

```diff
@@ -330,6 +336,35 @@ pub struct PegInOp {
     pub burn_header_hash: BurnchainHeaderHash, // hash of the burn chain block header
 }
 
+#[derive(Debug, PartialEq, Clone, Eq, Serialize, Deserialize)]
+pub struct PegOutRequestOp {
+    pub amount: u64,                 // sBTC amount to peg out, in satoshis
+    pub recipient: PoxAddress,       // Address to receive the BTC when the request is fulfilled
+    pub signature: MessageSignature, // Signature from sBTC owner as per SIP-021
+    pub memo: Vec<u8>,               // extra unused byte
```

Comment:
As discussed earlier this week, this op is going to need to pay a nominal amount of BTC to the peg wallet.  Eventually, this is going to be used by the requester to pay the stackers the BTC transaction fee to process the peg-out fulfillment.  For now, this is going to be used to link the peg-out request to the peg-out fulfillment.

Please update this op to include both the peg wallet address (a `PoxAddress`) and the amount paid.  The amount paid must be positive.

Reply:
I've added the `peg_wallet_address` and `fulfillment_fee` fields to the op for this :heavy_check_mark: